symfony/plugins/orangehrmTimePlugin/dao/CustomerDao.php

```diff
@@ -71,12 +82,12 @@ public function getCustomerCount($activeOnly = true) {
     /**
      *
      * @param type $customerId
-     * @return type 
+     * @return type
      */
-    public function getCustomerById($customerId) {
-
+    public function getCustomerById($customerId)
+    {
```

Comment:
please add `// TODO` if you didn't convert

Reply:
done